<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionTargetSelectorDialog</class>
 <widget class="QWidget" name="ActionTargetSelectorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select target for script action</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="iconHolder">
     <property name="text">
      <string notr="true">TextLabel</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>&amp;Add</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="5">
    <widget class="QTabWidget" name="eventTypeTab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mediaTabA">
      <attribute name="title">
       <string>Media file Line &amp;A</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QListView" name="mediaListLineA"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mediaTabB">
      <attribute name="title">
       <string>Media file Line &amp;B</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QListView" name="mediaListLineB"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="lightTab">
      <attribute name="title">
       <string>&amp;Light transition</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListView" name="lightList"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="SequenceTab">
      <attribute name="title">
       <string>Seq. entries</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QListView" name="sequenceEntryList"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>&amp;Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lineLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
